tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/Setup/PackageContentTests.ProjectSystem.verified.txt

```diff
@@ -487,36 +487,6 @@
   ProjectSystem/Rules/Items/ProjectItemsSchema.FSharp.tr.xaml,
   ProjectSystem/Rules/Items/ProjectItemsSchema.FSharp.zh-Hans.xaml,
   ProjectSystem/Rules/Items/ProjectItemsSchema.FSharp.zh-Hant.xaml,
-  AnalyzerReference.cs,
-  ResolvedAnalyzerReference.cs,
-  AssemblyReference.cs,
-  ResolvedAssemblyReference.cs,
-  COMReference.cs,
-  ResolvedCOMReference.cs,
-  FrameworkReference.cs,
-  ResolvedFrameworkReference.cs,
-  PackageReference.cs,
-  SuggestedWorkload.cs,
-  ResolvedPackageReference.cs,
-  ProjectReference.cs,
-  ResolvedProjectReference.cs,
-  SdkReference.cs,
-  ResolvedSdkReference.cs,
-  Compile.cs,
-  Content.cs,
-  Folder.cs,
-  None.cs,
-  GeneralConfiguredBrowseObject.cs,
-  DotNetCliToolReference.cs,
-  GeneralBrowseObject.cs,
-  ProjectDebugger.cs,
-  SupportedTargetFramework.cs,
-  SupportedNETCoreAppTargetFramework.cs,
-  SupportedNETStandardTargetFramework.cs,
-  SupportedNETFrameworkTargetFramework.cs,
-  SdkSupportedTargetPlatformIdentifier.cs,
-  SdkSupportedTargetPlatformVersion.cs,
-  VisualBasic.NamespaceImport.cs,
```

Comment:
The `PackageContentTests` unit test failed (as expected) after updating the CPS version, because these `.cs` files were not present in the VSIX. Removing them from this text file set a new baseline, and causes the test to pass.